package.json

```diff
@@ -53,7 +53,8 @@
   },
   "resolutions": {
     "@overnightjs/logger": "1.2.1",
-    "colors": "1.3.3"
+    "colors": "1.3.3",
+    "eth-sig-util": "3.0.1"
```

Comment:
You need to add this resolution to the package where it is being used as well.

Example: https://github.com/hoprnet/hoprnet/blob/master/packages/hoprd/package.json#L89

The reason is that this top-level resolution is only used while working in the monorepo and is unavailable when installing the packaged version. Thus, it must also be part of the package's resolutions.

Reply:
Gotcha, thanks for pointing it out, will change.

Reply:
Updated in [2967dda](https://github.com/hoprnet/hoprnet/pull/3384/commits/2967ddadfee4482f7e76446858be7a2adc1a3929)